solr/core/src/java/org/apache/solr/update/processor/TimeRoutedAliasUpdateProcessor.java

```diff
@@ -94,13 +92,15 @@
   private final SolrCmdDistributor cmdDistrib;
   private final CollectionsHandler collHandler;
   private final SolrParams outParamsToLeader;
+  @SuppressWarnings("FieldCanBeLocal")
   private final CloudDescriptor cloudDesc;
 
   private List<Map.Entry<Instant, String>> parsedCollectionsDesc; // k=timestamp (start), v=collection.  Sorted descending
   private Aliases parsedCollectionsAliases; // a cached reference to the source of what we parse into parsedCollectionsDesc
   private SolrQueryRequest req;
+  private ExecutorService preemptiveCreationExecutor;
```

Comment:
Since it will be nulled out in another thread, we ought to declare this as volatile.  I know this is being a bit pedantic since I don't think it'd be a real problem.